src/assets/translations/en/main.json

```diff
@@ -2585,7 +2586,25 @@
   "markets": {
     "recommended": "Recommended",
     "watchlist": "My Watchlist",
-    "categories": "Categories",
-    "marketsBody": "Nunc vel efficitur ligula, vel sagittis est. Morbi et sapien a ligula venenatis consequat."
+    "categoriesTabTitle": "Categories",
+    "marketsBody": "Nunc vel efficitur ligula, vel sagittis est. Morbi et sapien a ligula venenatis consequat.",
```

Comment:
👀

Reply:
Just following the Figma spec haha
<img width="1018" alt="image" src="https://github.com/user-attachments/assets/fa01d58f-ed88-4702-b145-3423465951e3">